frontend/src/scenes/plugins/plugin/DeliveryRateBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import { Badge } from 'antd'
+import { Tooltip } from 'lib/components/Tooltip'
+import { urls } from 'scenes/urls'
+import { Link } from 'lib/components/Link'
+
+type BadgeColor = 'green' | 'yellow' | 'red' | 'grey'
+
+export function DeliveryRateBadge({
+    deliveryRate,
+    pluginConfigId,
+}: {
+    deliveryRate: number | null
+    pluginConfigId: number
+}): JSX.Element {
+    const [color, tooltip] = deliveryRateSummary(deliveryRate)
+    return (
+        <Tooltip title={tooltip}>
+            <Link to={urls.appMetrics(pluginConfigId)}>
+                <Badge color={color} />
+            </Link>
+        </Tooltip>
+    )
+}
+
+function deliveryRateSummary(deliveryRate: number | null): [BadgeColor, string] {
+    if (deliveryRate === null) {
+        return ['grey', 'No events processed by this app in the past day']
```

Comment:
lets say `past 24h`, because in trends/graphs if you select today that's different from last 24h and that could be confusing here.